tokens/src/components/Popover.json

```diff
@@ -1,21 +1,14 @@
 {
-  "popover": {
-    "font": {
-      "size": { "value": "{font.size.sm.value}", "source": "$popover-font-size" }
-    },
-    "bg": { "value": "{color.background.base.value}", "source": "$popover-bg" },
-    "max-width": { "value": "480px", "source": "$popover-max-width" },
-    "border": {
-      "radius": { "value": "{size.border.radius.sm.value}", "source": "$popover-border-radius" },
-      "width": { "value": "{size.border.width.value}", "source": "$popover-border-width" },
-      "color": { "value": "rgba(0, 0, 0, .2)", "source": "$popover-border-color" }
-    },
-    "box-shadow": {
-      "value": "drop-shadow(0 2px 4px rgba(0, 0, 0, .15)) drop-shadow(0 2px 8px rgba(0, 0, 0, .15))",
-      "source": "$popover-box-shadow"
-    },
-    "header": {
-      "color": {
+  "typography": {
+    "popover": {
+      "font-size": { "value": "{font.size.sm.value}", "source": "$popover-font-size" }
+    }
+  },
+  "color": {
+    "popover": {
+      "bg": { "value": "{color.background.base.value}", "source": "$popover-bg" },
+      "border": { "value": "rgba(0, 0, 0, .2)", "source": "$popover-border-color" },
+      "header": {
         "base": { "value": "{headings.color.value}", "source": "$popover-header-color" },
```

Comment:
I think we should switch key to `text` here, because this color applies only to text I think

Reply:
Added